.vscode/launch.json

```diff
@@ -4,6 +4,13 @@
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
   "version": "0.2.0",
   "configurations": [
+    {
```

Comment:
This was added to help with debugging. It allows us to launch a go-nitro process in VSCode and set breakpoints.